BTDB/ODBLayer/RelationsInfo.cs

```diff
@@ -67,5 +67,13 @@ internal void LoadRelations(IEnumerable<KeyValuePair<uint, string>> relationName
                 if (name.Key >= _freeId) _freeId = name.Key + 1;
             }
         }
+
+        public IEnumerable<RelationInfo> EnumerateRelationInfos()
```

Comment:
why not return directly the Values collection?

Reply:
Good point. Fixed